test/new-e2e/scenarios/ndm/main.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/DataDog/datadog-agent/test/new-e2e/ndm/snmp"
+)
+
+func main() {
+	envNamePtr := flag.String("name", "snmp", "environment name")
+	destroyPtr := flag.Bool("destroy", false, "[optional] should destroy the environment")
```

Comment:
Note: We need to think how to start an env as having a `main` for each env doesn't scale well.

Reply:
We should probably have an interface for environments with a "CreateAndStart" and "Destroy" functions called by a common main, with the cool auto-discover you implemented in a previous PR in `test-infra`